src/quo2/components/markdown/list/style.cljs

```diff
@@ -0,0 +1,14 @@
+(ns quo2.components.markdown.list.style)
+
+(def container
+  {:flex-direction :row
+   :flex           1
+   :align-items    :flex-start
+   :margin-top     7})
+
+(def index
+  {:margin-left 5})
```

Comment:
I think the margin-left is 20 :thinking, and should it be a `:margin-horizontal`?

![image](https://github.com/status-im/status-mobile/assets/90291778/c1bb67ae-bec5-4cac-8286-feb0b4b50a66)

Reply:
> I think the margin-left is 20 :thinking, and should it be a `:margin-horizontal`?
> 
> ![image](https://user-images.githubusercontent.com/90291778/249308581-c1bb67ae-bec5-4cac-8286-feb0b4b50a66.png)

It's my opinion that we should not add these outer margins to the components directly but rather as they are being used as it leaves it more flexible to the page, otherwise you just have to fight with margins on use.

Reply:
I think when the designers use this component, it will always have that 20, but I'm not sure.

So I think your approach is ok